### ISSUE TITLE
Do not allow stalebot to close issues that have already been triaged.

### ISSUE THREAD

**Issue by douglasparker** [sct/overseerr#1845]
#### Description
Feature Requests seem to be continously marked as stale. This results in a lot of extra emails being sent to those of follow following issues and adds a lot of noise to our inbox. This also results in us literally fighting against stalebot and is extremely unproductive for everyone involved. 

#### Desired Behavior
Do not mark feature requests and other issues that have been triaged as stale.

#### Additional Context
n/a

**Comment by TheCatLady** [comment]
I disagree.  The whole point of stalebot is to make sure that there is still interest for a particular request.  It takes very minimal effort & time to comment when an issue is marked as stale, and doing so signals to the team that the request would not be a waste of our limited resources to implement.

You can disable GitHub email notifications if those are an issue.  I personally have those disabled, and review notifications on GitHub only.

**Comment by douglasparker** [comment]
I follow issues that I am interested in and would prefer a simple thumbs up rather than going back and forth. I personally will just let issues close because I can't be baby sitting every issue that I want to follow that uses stalebot on github.

I've seen other people keep replying to keep an important issue open, but over time especially with low priority issues people just get tired of it and stop, and then it gets closed and forgotten about.

You also have to remember that not everyone posting is a developer that can commit constantly bumping issues.

**Comment by TheCatLady** [comment]
Spoke with @sct about this... his plan is to introduce a new label for items that the team is _definitely_ going to be working on.  Those issues will be exempt from stalebot's monitoring.

However, feature requests without this label will still require a response every 2 months in order for them to not be closed by the bot.

If an issue is low priority and forgotten about, it probably is not worth our time.  Try to keep in mind that those who contribute to this project do so in their free time.  If you can't be bothered to comment on an issue, you also shouldn't expect us to spend our time working on it.

That said, if stalebot closes an issue you care about, you can always reach out o the team and let us know.  We can always re-open issues for which there is interest.

**Comment by douglasparker** [comment]
> Spoke with @sct about this... his plan is to introduce a new label for items that the team is _definitely_ going to be working on.  Those issues will be exempt from stalebot's monitoring.

I think this is a good compromise. 😁

**Comment by jyggen** [comment]
I'm kinda getting mixed messages here. Comments are required to show interest/prevent an issue from going stale, but at the same time the comments showing interest on the issues are marked as off-topic (e.g. #377). I even found a comment that was marked as off-topic that I'd argue is very much on-topic (last on in #1200).

**Comment by douglasparker** [comment]
> I'm kinda getting mixed messages here. Comments are required to show interest/prevent an issue from going stale, but at the same time the comments showing interest on the issues are marked as off-topic (e.g. #377). I even found a comment that was marked as off-topic that I'd argue is very much on-topic (last on in #1200).

Yeah, I'm not even making an effort to contribute in discussion here anymore.

It's actually infuriating to deal with, and I feel like it actively discourages community discussion and contribution. I still see important issues being marked as stale and I refuse to waste anymore time fighting against the bot or maintainers.

The software is great and they're doing the community a great service, however I do not agree with how issues are being handled on GitHub.

**Comment by sct** [comment]
> > I'm kinda getting mixed messages here. Comments are required to show interest/prevent an issue from going stale, but at the same time the comments showing interest on the issues are marked as off-topic (e.g. #377). I even found a comment that was marked as off-topic that I'd argue is very much on-topic (last on in #1200).
> 
> Yeah, I'm not even making an effort to contribute in discussion here anymore.
> 
> It's actually infuriating to deal with, and I feel like it actively discourages community discussion and contribution. I still see important issues being marked as stale and I refuse to waste anymore time fighting against the bot or maintainers.
> 
> The software is great and they're doing the community a great service, however I do not agree with how issues are being handled on GitHub.
> 
> Edit: Here is one single issue. It's absolutely ridiculous.
> 
> ![BM62xTN](https://user-images.githubusercontent.com/18235822/130489925-cf0bcdcc-e14b-481d-bf2d-4ded600ed97f.png)
> 
> This is not a productive workflow. Not only that, but I get an email each time for each issue I follow.

I get the frustration and as was already mentioned in this PR, this is something I want to fix. As you can probably tell by the lack of commits and releases for a while I haven't been around much. I've been wildly busy with a new job, a move, and other personal life bits.

I will try to get the ignore label in this week sometime, but another thing I am considering is just moving our feature requests to a different system because it's becoming a huge chore to juggle and keep track of them in GitHub.

**Comment by douglasparker** [comment]
I'm still constantly having to reply to stalebot with a single character to keep issues from being closed. This is tedious and doesn't contribute to the discussion at all. It simply adds noise to issues to keep them open, despite the community already voicing their interest.

Is there going to be a solution to this problem at some point? Perhaps moving feature requests off GitHub?

At this point, I am ready to unfollow all open issues because it's turned into a giant waste of my time.

I know that I am not the only one who despises the way stalebot is being used in this project.

@sct @TheCatLady

**Comment by douglasparker** [comment]
Stalebot is still extremely annoying with this project. I want to follow issues that I care about, but most of the time the emails I get are someone bumping an issue after it is marked as stale.

Using stalebot as a method of measuring user interest as previously mentioned is annoying at best. A simple emoji reaction is all we need.

I and many others use GitHub to be productive, and projects using stalebot like this one is anything but that.
